sockeye/config.py

```diff
@@ -119,7 +121,7 @@ def load(fname: str) -> 'Config':
         :return: Configuration.
         """
         with open(fname) as inp:
-            obj = yaml.load(inp)
+            obj = yaml.load(inp, Loader=yaml.UnsafeLoader)  # type: ignore
```

Comment:
Would it be better to use yaml_loader here?

Reply:
can you elaborate? I am not familiar with yaml that much.